tests/pickle-rcparams.py

```diff
@@ -0,0 +1,294 @@
+          u'mathtext.tt': u'monospace',
+          u'nbagg.transparent': True,
+          u'patch.antialiased': True,
+          u'patch.edgecolor': u'k',
+          u'patch.facecolor': u'b',
+          u'patch.linewidth': 1.0,
+          u'path.effects': [],
+          u'path.simplify': True,
+          u'path.simplify_threshold': 0.1111111111111111,
+          u'path.sketch': None,
+          u'path.snap': True,
+          u'pdf.compression': 6,
+          u'pdf.fonttype': 3,
+          u'pdf.inheritcolor': False,
+          u'pdf.use14corefonts': False,
+          u'pgf.debug': False,
+          u'pgf.preamble': [],
+          u'pgf.rcfonts': True,
+          u'pgf.texsystem': u'xelatex',
+          u'plugins.directory': u'.matplotlib_plugins',
+          u'polaraxes.grid': True,
+          u'ps.distiller.res': 6000,
+          u'ps.fonttype': 3,
+          u'ps.papersize': u'letter',
+          u'ps.useafm': False,
+          u'ps.usedistiller': False,
+          u'savefig.bbox': None,
+          u'savefig.directory': u'~',
+          u'savefig.dpi': 100.0,
+          u'savefig.edgecolor': u'w',
+          u'savefig.facecolor': u'w',
+          u'savefig.format': u'png',
+          u'savefig.frameon': True,
+          u'savefig.jpeg_quality': 95,
+          u'savefig.orientation': u'portrait',
+          u'savefig.pad_inches': 0.1,
+          u'savefig.transparent': False,
+          u'svg.fonttype': u'path',
+          u'svg.image_inline': True,
+          u'svg.image_noscale': False,
+          u'text.antialiased': True,
+          u'text.color': u'k',
+          u'text.dvipnghack': None,
+          u'text.hinting': u'auto',
+          u'text.hinting_factor': 8,
+          u'text.latex.preamble': [],
+          u'text.latex.preview': False,
+          u'text.latex.unicode': False,
+          u'text.usetex': False,
+          u'timezone': u'UTC',
+          u'tk.window_focus': False,
+          u'toolbar': u'toolbar2',
+          u'verbose.fileo': u'sys.stdout',
+          u'verbose.level': u'silent',
+          u'webagg.open_in_browser': True,
+          u'webagg.port': 8988,
+          u'webagg.port_retries': 50,
+          u'xtick.color': u'k',
+          u'xtick.direction': u'in',
+          u'xtick.labelsize': u'medium',
+          u'xtick.major.pad': 4.0,
+          u'xtick.major.size': 4.0,
+          u'xtick.major.width': 0.5,
+          u'xtick.minor.pad': 4.0,
+          u'xtick.minor.size': 2.0,
+          u'xtick.minor.visible': False,
+          u'xtick.minor.width': 0.5,
+          u'ytick.color': u'k',
+          u'ytick.direction': u'in',
+          u'ytick.labelsize': u'medium',
+          u'ytick.major.pad': 4.0,
+          u'ytick.major.size': 4.0,
+          u'ytick.major.width': 0.5,
+          u'ytick.minor.pad': 4.0,
+          u'ytick.minor.size': 2.0,
+          u'ytick.minor.visible': False,
+          u'ytick.minor.width': 0.5}
+
+pickle.dump(rcparams, open('rcparams.pkl', 'wb'))
```

Comment:
I'm a bit confused about this. Why are we including this file to dump the default ``rcParams``? And a pickle for it as well?